src/index.tsx

```diff
@@ -1,25 +1,28 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './style.scss';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
 import * as serviceWorker from './serviceWorker';
 import { Provider } from 'react-redux';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
 
-import promise from 'redux-promise';
+import './style.scss';
+import promise from 'redux-promise'
 import reducers from './reducers';
-import { applyMiddleware, createStore } from 'redux';
+import { applyMiddleware, createStore, compose } from 'redux';
+import thunk from 'redux-thunk';
 
 import BeerList from './containers/beerList';
 import AddNew from './containers/addNew';
 
-const createStoreWithMiddleware = applyMiddleware(promise)(createStore);
+const middleware = [ promise, thunk ];
```

Comment:
AFAIK promise and thunk do the same thing

Reply:
I was testing thunk, forgot to delete it after